### DISCUSSION TITLE
Very strange performance behavior on mid-end pc

### DISCUSSION THREAD

**Discussion by doesntmattervoid** [ppy/osu#27049] [Q&A]
Having statters, omega lag on some maps, cursor stucks on spinners sometime(pw3395, so its impossible, have the same problem on my tablet, in stable - works good as always) also, sometimes notes wont reg, but the z/x key was pressed.
If you know whats the issue, I would be very grateful for the solution.
PC specs - I3 7100, RX 580, 8 gb ram, windows 10 ghost spectre 20h2
[1707137832.runtime.log](https://github.com/ppy/osu/files/14166396/1707137832.runtime.log)
[1707137832.performance.log](https://github.com/ppy/osu/files/14166398/1707137832.performance.log)
[1707137832.updater.log](https://github.com/ppy/osu/files/14166399/1707137832.updater.log)
[1707137832.input.log](https://github.com/ppy/osu/files/14166400/1707137832.input.log)
[1707137832.network.log](https://github.com/ppy/osu/files/14166401/1707137832.network.log)

**Comment by Joehuu** [comment]
Maybe this is the same issue as https://github.com/ppy/osu/issues/26946. Can you try disabling the setting mentioned there or change renderer and see if there's an improvement?

**Reply by doesntmattervoid** [reply]
Sync is disabled, so no

**Reply by doesntmattervoid** [reply]
on opengl having the same issue

**Reply by Stoppedpuma** [reply]
For performance start here: https://osu.ppy.sh/wiki/en/Performance_troubleshooting

A few off of the top of my head though could be out of date drivers, programs running in the background (discord is a big one I hear), possible thermal throttling? But what mostly sticks out to me is "windows 10 ghost spectre 20h2", now yes custom builds of windows may have performance improvements though not very large and can be a huge cause of issues, especially ones nearly 4 years old.

Regarding inputs:

Does your framerate cap when your inputs drop? if so your game might just be tabbing out for some reason? Another good thing to try is toggling raw input on / off and seeing if that helps. And probably the least likely but some what similar thing is #25465 but this is more likely a linux specific issue.

**Reply by doesntmattervoid** [reply]
tried on normal default windows 11, the same thing. I think maybe the reason is cpu, or maybe gpu. Tried stable drivers and also a pro drivers for rendering 3d graphics, the same performance.

**Reply by Stoppedpuma** [reply]
Try seeing if temperatures could be the problem, MSI Afterburner should work for monitoring. Is your tablet still having issues?

**Reply by doesntmattervoid** [reply]
so, i have 45 on cpu, 50 on gpu. very cold

**Reply by doesntmattervoid** [reply]
tablet is fixed, thx for the solution. I mean, i just turned off opendriver for tablet

**Reply by Stoppedpuma** [reply]
Have you tried using vulkan? Bit of a hidden for now but you can edit the framework.ini inside your osulazer folder and change the `Renderer = whateveryoureusing` to `Renderer = Vulkan`

**Reply by doesntmattervoid** [reply]
no more framework.ini i think

**Reply by Stoppedpuma** [reply]
framework.ini still exists, I think you're just lost in your file system? I believe the lazer folder by default on windows is c:/users/yourname/appdata/local/osulazer? Also your issue may be related to #26998 I'm unable to reproduce this behavior on linux but majority of the people with this issue are on windows.

**Comment by doesntmattervoid** [comment]
You was right, thanks!

вт, 5 мар. 2024 г., 23:39 Stoppedpuma ***@***.***>:
